java-squid/src/test/files/se/UnclosedResourcesCheck.java

```diff
@@ -240,4 +245,16 @@ public void correctHandlingOfJarFile(String fileName) {
       closeJar(jar);
     }
   }
+
+  public void getDirectivesFromFile(File aFile) {
```

Comment:
This sounds unconnected to your refactoring and should not be part of this commit.